src/distributors/distributor-factory.ts

```diff
@@ -1,12 +1,12 @@
-import { AdoptOpenJDKDistributor } from './adoptopenjdk-installer';
+import { AdoptOpenJDKDistributor } from './adoptium/adoptium-installer';
 import { JavaBase } from './base-installer';
 import { JavaInstallerOptions } from './base-models';
 import { LocalDistributor } from './local-installer';
-import { ZuluDistributor } from './zulu-installer';
+import { ZuluDistributor } from './zulu/zulu-installer';
 
 // TO-DO: confirm distributor names
 enum JavaDistributor {
-  AdoptOpenJdk = 'adoptOpenJdk',
+  AdoptOpenJdk = 'adoptium',
```

Comment:
Could you please also create item in enum for `jdkFile`

Reply:
Done. Thank you.